phpcs.xml.dist

```diff
@@ -21,25 +21,14 @@
     <arg name="report" value="source"/>
 
     <!-- PHP Rules -->
-    <rule ref="PHPCompatibility" />
-    <config name="testVersion" value="5.2-"/>
-    <rule ref="Squiz.PHP.DisallowMultipleAssignments.Found">
-        <severity>0</severity>
+    <rule ref="PHPCompatibility">
+        <exclude-pattern>*/tests/*</exclude-pattern>
```

Comment:
Worth adding a comment as to why this is excluded. 
If it's because the tests have a higher minimum version of PHP, then you can take one of two approaches.

1. Add a `phpcs.xml.dist` file inside the the tests directory, then add a second call to phpcs the composer script referencing this config file
2. Just add a runtime config value to the composer script. See [this](https://github.com/GaryJones/plugin-boilerplate/blob/6a0515058bbb59826b3be190f1930a2dea016982/composer.json#L58) as an example, which happens to test that my main plugin file is 5.2 compatible, but you can easily see how you could just check that your tests/ directory is 5.* compatible.